modules/real_time_enforcer/files/policy/bigquery/common.rego

```diff
@@ -1,3 +1,18 @@
+# Copyright 2019 The micromanager Authors. All rights reserved.
```

Comment:
Who are the micromanager authors?

If we want to reference a set of multiple authors, we need an authors file: https://g3doc.corp.google.com/company/teams/opensource/releasing/authors.md?cl=head

Reply:
AUTHORS file here:  https://github.com/cleardataeng/micromanager/blob/master/AUTHORS

Where should this file go?

Reply:
The submodule root.

Reply:
@morgante Do we need to include Forseti and CFT as authors as well?

Reply:
Added the AUTHORS file with micromanager authors.

Reply:
Also need `Google LLC` as an author.